### ISSUE TITLE
TypeError while printing out accessibility violations

### ISSUE THREAD

**Issue by adams0619** [pa11y/pa11y-ci#50]
Since upgrading to the latest `2.0.0` release of `pa11y-ci` I get the following error whenever `pa11y-ci` is run using a standard other than `WCAG2AA` (the default). While `pa11y-ci` runs through the tests just fine, the issue seems to come from one of the dependencies when printing out errors from the test run as seen below. 

**Note:** This errors does not seem to occur when I run the regular non-ci centric `pa11y` 

_Environment:_
```
Pa11y:      5.0.3
Node.js:    8.9.4
npm:        5.6.0
OS:         16.7.0 (darwin)
```
```
> pa11y-ci -V
2.0.0
```

Error:
-- 

```
> pa11y-ci --config tests/pa11y/pa11y.json

Running Pa11y on 1 URLs:
 > http://localhost:9090/services - 2 errors

Errors in http://localhost:9090/services:

 • The heading structure is not logically nested. This h2 element appears to be the primary document heading, so should be an h1 element.

   (#burger-menu-container > div:nth-child(2) > ul > div > h2)

   <h2><a href="javascript:void();">Pr...</h2>
   
/Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/node_modules/async/asyncify.js:108
    throw error;
    ^

TypeError: Cannot read property 'replace' of null
    at report.results.(anonymous function).forEach.result (/Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/lib/pa11y-ci.js:136:40)
    at Array.forEach (native)
    at Object.keys.forEach.url (/Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/lib/pa11y-ci.js:125:27)
    at Array.forEach (native)
    at Object.testRunComplete [as drain] (/Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/lib/pa11y-ci.js:122:33)
    at /Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/node_modules/async/internal/queue.js:113:19
    at /Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/node_modules/async/internal/onlyOnce.js:12:16
    at invokeCallback (/Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/node_modules/async/asyncify.js:101:9)
    at /Users/macbookpro/work/project-tr12/node_modules/pa11y-ci/node_modules/async/asyncify.js:89:17
    at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by rowanmanning** [comment]
Hmm, that's not great. Taking a look now :)

**Comment by rowanmanning** [comment]
Hey @adams0619. I think this should be fixed, I just released version `2.0.1`. Could you try it out and confirm, please?

**Comment by adams0619** [comment]
@rowanmanning Just updated and it's fixed. Thank you so much for looking at this and for the awesome testing library.

**Comment by rowanmanning** [comment]
No worries :) glad to help
